tooltester/src/main/java/io/dockstore/tooltester/BlackList.java

```diff
@@ -0,0 +1,36 @@
+package io.dockstore.tooltester;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import io.dockstore.tooltester.Models.BlackListObject;
+
+/**
+ * These are Tool IDs and ToolVersion names that are known to always fail on the latest Dockstore CLI
+ * @author gluu
+ * @since 04/04/19
+ */
+public class BlackList {
+    public static final List<BlackListObject> list = Collections.unmodifiableList(
+            new ArrayList<BlackListObject>() {{
```

Comment:
I wouldn't change this, but you could have used `Arrays.asList` and saved yourself all the `add`s. Just a personal preference FYI; nothing wrong with the way you are doing.